Meadowlark.net/Meadowlark.Net.Core/Meadowlark.Net.Core.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
```

Comment:
Glad to see this already present. Making nullability an opt-in instead of always-on is a great new-ish feature in C#.